File: iteration2/src/CourseSession.java
import java.util.ArrayList;

public class CourseSession {

    private ArrayList<String> courseDay;
    private ArrayList<String> courseStartTime;
    private ArrayList<String> courseEndTime;
    private ArrayList<String> coursePlace;

    public CourseSession(ArrayList<String> courseDay, ArrayList<String> courseStartTime, ArrayList<String> courseEndTime, ArrayList<String> coursePlace) {
        this.courseDay = courseDay;
        this.courseStartTime = courseStartTime;
        this.courseEndTime = courseEndTime;
        this.coursePlace = coursePlace;
    }

    public ArrayList<String> getCourseDay() {
        return courseDay;
    }

    public void setCourseDay(ArrayList<String> courseDay) {
        this.courseDay = courseDay;
    }

    public ArrayList<String> getCourseStartTime() {
        return courseStartTime;
    }

    public void setCourseStartTime(ArrayList<String> courseStartTime) {
        this.courseStartTime = courseStartTime;
    }

    public ArrayList<String> getCourseEndTime() {
        return courseEndTime;
    }

    public void setCourseEndTime(ArrayList<String> courseEndTime) {
        this.courseEndTime = courseEndTime;
    }

    public ArrayList<String> getCoursePlace() {
        return coursePlace;
    }

    public void setCoursePlace(ArrayList<String> coursePlace) {
        this.coursePlace = coursePlace;
    }
}

File: iteration3/src/course_session.py
class CourseSession:
    def __init__(self, course_day, course_start_time, course_end_time, course_place):
        self.__course_day = course_day
        self.__course_start_time = course_start_time
        self.__course_end_time = course_end_time
        self.__course_place = course_place

    def get_course_day(self):
        return self.__course_day

    def set_course_day(self, course_day):
        self.__course_day = course_day

    def get_course_start_time(self):
        return self.__course_start_time

    def set_course_start_time(self, course_start_time):
        self.__course_start_time = course_start_time

    def get_course_end_time(self):
        return self.__course_end_time

    def set_course_end_time(self, course_end_time):
        self.__course_end_time = course_end_time

    def get_course_place(self):
        return self.__course_place

    def set_course_place(self, course_place):
        self.__course_place = course_place
